bundles/admin/admin-layereditor/view/AdminLayerForm.jsx

```diff
@@ -17,20 +17,25 @@ const AdminLayerForm = ({
     mapLayerGroups,
     dataProviders,
     layer,
-    message = {},
+    messages = [],
     onCancel,
     onDelete,
     onSave,
     getMessage
 }) => {
-    if (message.key) {
-        message.text = getMessage(message.key);
-    }
+    const mappedMessages = [];
+    messages.forEach(m => {
+        if (m.key) {
+            m.text = getMessage(m.key);
+        }
+        if (m.text) {
+            mappedMessages.push(<Alert key={m.key} message={m.text} type={m.type} />);
```

Comment:
Maybe add some margin to Alert since there can now be several messages?

Reply:
Added margin-bottom with 5 pixels